prow/plugins/lgtm/lgtm.go

```diff
@@ -41,13 +41,17 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The Config field is omitted because this plugin is not configurable.
-	return &pluginhelp.PluginHelp{
-			Description: "The lgtm plugin manages the application and removal of the 'lgtm' (Looks Good To Me) label which is typically used to gate merging.",
-			WhoCanUse:   "Members of the organization that owns the repository. '/lgtm cancel' can be used additionally by the PR author.",
-			Usage:       "/lgtm [cancel]",
-			Examples:    []string{"/lgtm", "/lgtm cancel"},
-		},
-		nil
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The lgtm plugin manages the application and removal of the 'lgtm' (Looks Good To Me) label which is typically used to gate merging.",
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/lgtm [cancel]",
+		Description: "'lgtm' a PR which is typically used to gate merging.",
```

Comment:
Adds or removes the `lgtm` label which is typically used to gate merging.

Reply:
done